Configuration/Entitlements/App-ios.entitlements

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.carplay-driving-task</key>
```

Comment:
this needs to be injected via the `Configuration/Entitlements/activate_special_entitlements.sh` script, so other developers (without the entitlement) can use it.

Reply:
<img width="799" alt="Screenshot 2023-11-29 at 21 48 26" src="https://github.com/home-assistant/iOS/assets/5808343/176ecc28-4ff4-4ba4-adc4-91bffa9b3c77">
This is an example I just did for Thread network credentials entitlement @LuisFALopes

Reply:
<img width="1701" alt="Screenshot 2023-11-29 at 21 49 50" src="https://github.com/home-assistant/iOS/assets/5808343/51e36960-81a7-4e36-9fb1-ded109ea9bd5">
And also this